frontend/app-development/features/policyEditor/SettingsModal/SettingsModal.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import classes from './SettingsModal.module.css';
+import { Button, Heading } from '@digdir/design-system-react';
+import { useTranslation } from 'react-i18next';
+import { CogIcon } from '@navikt/aksel-icons';
+import { Modal } from 'app-shared/components/Modal';
+
+/**
+ * Displays the settings modal. It displays the content inside the Modal component.
+ *
+ * @returns {React.ReactNode}
+ */
+export const SettingsModal = (): React.ReactNode => {
+  const { t } = useTranslation();
+
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <>
+      {/* TODO - Move button to the correct place to open the modal from. Issue: #11047 */}
+      <Button onClick={() => setIsOpen(true)}>{t('settings_modal.open_button')}</Button>
+      <Modal
```

Comment:
Trenger vi å legge til aria-propen for å gjøre modalen mer tilgjengelig? 🤔 

https://reactcommunity.org/react-modal/accessibility/#aria